discoverx/dx.py

```diff
@@ -49,7 +49,9 @@ def __init__(
 
     def _can_read_columns_table(self) -> bool:
         try:
-            self.spark.sql(f"SELECT * FROM {self.INFORMATION_SCHEMA}.columns WHERE table_catalog = 'system' LIMIT 1")
+            self.spark.sql(
+                f"SELECT * FROM {self.INFORMATION_SCHEMA}.columns WHERE table_catalog = 'system' AND table_schema = 'columns' LIMIT 1"
```

Comment:
shouldn't it be table_schema= 'information_schema'

Reply:
Yes indeed. I updated it.
It actually works any way because we don't care about the result, but just that there is no exception.